### PR TITLE
fix(sync): #MC-243 add default TTL

### PR DESCRIPTION
**Pull request by alicedraillard**
## Issue ticket number and link
[MC-248](https://jira.support-ent.fr/browse/MC-248)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/java/net/atos/entng/calendar/helpers/CalendarHelper.java [hunk 1]
```diff
@@ -55,6 +55,9 @@ public Future<Void> externalCalendarSync(String calendarId, UserInfos user, Stri
                     if (!params.isEmpty()) { //set the calendar to not updating if there is a fail
                         updateCalendar(params.getJsonObject(Field.CALENDAR).put(Field.ISUPDATING, false));
                     }
+                    String message = String.format("[Calendar@%s::externalCalendarSync]:  an error has occurred during " +
```
Comment (alicedraillard) [inline src/main/java/net/atos/entng/calendar/helpers/CalendarHelper.java:58]: add error message (used while debugging this ticket)

File: src/main/java/net/atos/entng/calendar/helpers/CalendarHelper.java [hunk 2]
```diff
@@ -173,17 +176,17 @@ private Future<Void> updateCalendar(JsonObject calendar) {
...
      * @return {@link Boolean} true if the time since the last update is longer than the minimum time between two updates
      */
     public Boolean isTimeToLivePast(JsonObject calendar) {
         Timestamp lastUpdateTimestamp = new Timestamp(calendar.getJsonObject(Field.UPDATED).getLong(MongoField.$DATE));
         Date lastUpdateDate = new Date(lastUpdateTimestamp.getTime());
-
         long secondsSinceLastUpdate = (new Date().getTime()-lastUpdateDate.getTime())/1000;
 
-        return secondsSinceLastUpdate - config.getLong(Field.CALENDARSYNCTTL) > 0;
+        return secondsSinceLastUpdate - config.getLong(Field.CALENDARSYNCTTL, 900L) > 0;
```
Comment (qpic-cgi) [inline src/main/java/net/atos/entng/calendar/helpers/CalendarHelper.java:189]: 900L ne peux pas être en config ?

**Review by qpic-cgi — APPROVED** [review]

File: src/main/java/net/atos/entng/calendar/helpers/CalendarHelper.java (hunk 2 above)
Comment (alicedraillard) [reply src/main/java/net/atos/entng/calendar/helpers/CalendarHelper.java:189]: justement c'est au cas où on n'arrive pas à récupérer la config

**Review by JinDouang — APPROVED** [review]

File: src/main/java/net/atos/entng/calendar/helpers/CalendarHelper.java (hunk 2 above)
Comment (qpic-cgi) [reply src/main/java/net/atos/entng/calendar/helpers/CalendarHelper.java:189]: Oops ! pardon 😝

**Pull request merged**